Fixes #42319, closes bold tag for IRC admin pms

senjx opened the pull request:
Adds a missing closing bold tag to fix the formatting of IRC admin PMs.
Fixes https://github.com/tgstation/tgstation/issues/42319

senjx committed:
Closes missing bold tag

PKPenguin321 commented:
Put `Fixes https://github.com/tgstation/tgstation/issues/42319` in your PR body to autoclose the issue when this PR is merged

senjx committed:
Adds force and flag checks for items attacking brains

Adds a force check and a NOBLUDGEON flag check for items that attack brains to prevent accidental or ludicrous brain damage via straw.

senjx commented:
I'm a retard and forgot to make a separate fork for a separate pull

PKPenguin321 commented:
It happens, don't say the R word here though or github will delete you
